concurrency/spi/src/main/java/org/jboss/as/ee/concurrent/ConcurrentContext.java

```diff
@@ -210,7 +210,8 @@ private void readObject(ObjectInputStream in) throws IOException, ClassNotFoundE
             // switch to EE module classloader, otherwise, due to serialization, deployments would need to have dependencies to other internal modules
             final Module module;
             try {
-                module = Module.getBootModuleLoader().loadModule(ModuleIdentifier.create("org.jboss.as.ee"));
+                String canonicalModuleName = ModuleIdentifierUtil.canonicalModuleIdentifier("org.jboss.as.ee");
```

Comment:
It is not necessary to use ModuleIdentifierUtil.canonicalModuleIdentifier @RanabirChakraborty  here because "org.jboss.as.ee" is standard module name.

Reply:
Okay, but I'm unsure what to use here to replace `ModuleIdentifier`. If we want to replace it with `ModuleIdentifierUtil` then I guess it's the only way. If you have anything else in your mind, please suggest.

Reply:
@RanabirChakraborty it's simply

module = Module.getBootModuleLoader().loadModule("org.jboss.as.ee");

The key thing I wrote ModuleIdentifierUtil.canonicalModuleIdentifier for was to ensure arbitrary module name strings that external users provide are in the canonical form. This isn't that; it's a static string the WF devs wrote, in the canonical form. It doesn't need to be canonicalized.

An incorrect form would be "org.jboss.as.ee:main", which an external user might think is correct, but isn't.